extensions/functions_arithmetic.yaml

```diff
@@ -181,6 +211,72 @@ scalar_functions:
           - value: i64
           - value: i64
         return: i64
+  -
+    name: "power"
+    description: >
+      Take the power with the first value as the base and second as exponent.
+
+      All integer arguments are promoted to int64.
```

Comment:
Given that this is the behavior, let's not include the variations that take anything else. Let's just expect engines to do upromotions.